package.json

```diff
@@ -980,6 +887,16 @@
           "type": "string",
           "description": "%config.default_namespace%"
         },
+        "i18n-ally.translate.deeplApiKey": {
+          "type": "string",
+          "default": null,
+          "description": "%config.deepl_api_key%"
+        },
+        "i18n-ally.deeplLog": {
```

Comment:
I am thinking maybe we should have these?

```
"i18n-ally.translate.deepl.apiKey"
"i18n-ally.translate.deepl.enabledLog"
```

As we may have these in the future

```
"i18n-ally.translate.google.apiKey"
"i18n-ally.translate.xxxx.apiKey"
```

Reply:
that's sound great